src/pieces/logs/serverLog.ts

```diff
@@ -309,6 +310,41 @@ async function processMessageDelete(msg: Message | PartialMessage, serverLog: Te
 	}
 
 	serverLog.send({ embeds: [embed], files: attachments });
+
+	const bot = msg.client;
+	handleExpDetract(bot, msg);
+}
+
+async function handleExpDetract(bot: Client, msg: Message | PartialMessage) {
+	const user: SageUser = await msg.author.client.mongo.collection(DB.USERS).findOne({ discordId: msg.author.id });
+	console.log(user);
+	// bot.mongo.collection(DB.USERS).findOneAndUpdate(
+	// 	{ user },
+	// 	{ $inc: { count: -1, curExp: +1 } }
+	// );
```

Comment:
These lines probably shouldn't be here anymore

Reply:
Forgot about that heh, will be removed in the next commit